airbyte-webapp/src/components/ui/ListBox/ListBox.tsx

```diff
@@ -17,9 +17,16 @@ export interface Option<T> {
   value: T;
 }
 
+export interface Footer<T> {
+  label: string;
+  value: T;
+  link: string;
+}
+
 interface ListBoxProps<T> {
   className?: string;
   options: Array<Option<T>>;
+  footer?: Footer<T>;
```

Comment:
Not sure if the footer needs its own interface: I imagine it'll be the same as an option, but it'll need a link.